packages/cli/src/lib/app-config/types.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type AppConfig = any;
```

Comment:
`Record<string, any>` perhaps? :) or is it time for these JsonThing types?

Reply:
Yeah `JsonObject` is on it's way, and it's probably gonna be something like.

```ts
type AppConfg = {
  global?: JsonObject
  packages?: { [name in string]: JsonObject }
}
```

And maybe even a few included config keys.

The `AppConfig` type is just a placeholder for now, to make it easier to refactor later without forgetting to remove `any`'s